docs/reference/snapshot-restore/apis/delete-snapshot-api.asciidoc

```diff
@@ -30,6 +30,7 @@ PUT /_snapshot/my_repository/snapshot_3?wait_for_completion=true
 -----------------------------------
 DELETE /_snapshot/my_repository/my_snapshot
 -----------------------------------
+// TEST[s/my_snapshot/my_snapshot?wait_for_completion=false/]
```

Comment:
I'd rather we tested that this param works more explicitly in a YAML test rather than so implicitly here.

Reply:
I added a rudimentary YAML test (please see PR and musings in team channel about approach). I have left this one in here, I'm not entirely certain whether that's the right thing to do, I thought that maybe it's useful for testing that the doc is in sync with the implementation?